### PR TITLE
feat: Expose SentryDebugImageProvider

### PR DESCRIPTION
**Pull request by philipphofmann**
## :scroll: Description

Expose SentryDebugImageProvider for hybrid SDKs.

## :bulb: Motivation and Context

Fixes GH-1089

## :green_heart: How did you test it?
CI.

## :pencil: Checklist

- [x] I reviewed the submitted code
- [x] I added tests to verify the changes
- [x] I updated the CHANGELOG if needed
- [x] I updated the docs if needed
- [x] Review from the native team if needed
- [x] No breaking changes

### CODE REVIEW

File: Sources/Sentry/Public/SentryDebugMeta.h [hunk 1]
```diff
@@ -4,6 +4,13 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
+/**
+ * This class is actually a DebugImage:
+ * https://develop.sentry.dev/sdk/event-payloads/debugmeta/#debug-images and should be renamed to
```
Comment (bruno-garcia) [inline Sources/Sentry/Public/SentryDebugMeta.h:9]: If this was internal isn't it safe to rename now?

Comment (philipphofmann) [reply Sources/Sentry/Public/SentryDebugMeta.h:9]: This was already public see: https://github.com/getsentry/sentry-cocoa/blob/dd880f3da9884282cd0cde294af6118638b275cb/Sources/Sentry/Public/Sentry.h#L13.

The SentryDebugImageProvider was not public before.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,8 +1,9 @@
 # Changelog
 
-## Unreleased 
+## Unreleased
 
-docs: Improve code doc on start and endSession (#1098)
+- feat: Expose SentryDebugImageProvider (#1094)
+- docs: Improve code doc on start and endSession (#1098)
```
Comment (marandaneto) [inline CHANGELOG.md:6]: is this part of this PR?

**Review by marandaneto — APPROVED** [review]
left a comment but all good, LGTM

File: CHANGELOG.md (hunk 2 above)
Comment (philipphofmann) [reply CHANGELOG.md:6]: This stems from merging master into this branch and resolving conflicts. So we are good to go.

**Pull request merged**